api/bin/index.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
```

Comment:
I wonder about versioning the generated JS files from our TypeScript? I think that I would usually not version control compiled versions of other source files. Should we `.gitignore` `api/bin/*.js` too?

Reply:
Good catch! I'll make that change now.